taichi/python/export_ggui.cpp

```diff
@@ -232,6 +238,7 @@ struct PyScene {
     renderable_info.draw_first_index = (int)draw_first_index;
     renderable_info.draw_vertex_count = (int)draw_vertex_count;
     renderable_info.draw_first_vertex = (int)draw_first_vertex;
+    renderable_info.display_mode = taichi::lang::PolygonMode(display_mode);
```

Comment:
Yea this is really error prone(and why you should just expose PolygonMode enum to python instead) : imagine what if you inserted a new entry in the beginning of PolygonMode...

Reply:
Fixed that.